From 2ad0fd8ab07e54f5cdf66b770c8f1155cb469247 Mon Sep 17 00:00:00 2001
From: Dima Barsky <dima@debian.org>
Date: Tue, 29 Mar 2016 19:23:26 +0200
Subject: import inspect in urllib 2

 Added "import inspect" to M2Crypto/m2urllib2.py (Closes: #493314, #484364, #477799).
Forwarded: no
Reviewed by: Daniel Stender <stender@debian.org>
Last-Update: 2017-11-05
---
 M2Crypto/m2urllib2.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/M2Crypto/m2urllib2.py b/M2Crypto/m2urllib2.py
index 0596c1c..49b6422 100644
--- a/M2Crypto/m2urllib2.py
+++ b/M2Crypto/m2urllib2.py
@@ -14,6 +14,7 @@ Summary of changes:
 """
 
 import socket
+import inspect
 
 from M2Crypto import SSL, httpslib, six, util
 
